ballerina-tests/tests/test_services.bal

```diff
@@ -742,6 +742,34 @@ service /special_types on specialTypesTestListener {
     resource function get company() returns Company {
         return company;
     }
+
+    resource function get month(Month month) returns string {
```

Comment:
Isn't it necessary to return the Month in order to verify the bug fix?

Reply:
Yes, otherwise the passed string would be `JANUARY`, `February`, etc.

Reply:
If we return the `Month` type here(just like the example in the issue), we can verify the output value as well.

Reply:
Right. I think you are correct. Will update the code.

Reply:
Fixed with 008f37bb942e035068628ce8a344c0ff201ddfe7